docs/tools/erc725js/providers.md

```diff
@@ -8,16 +8,12 @@ The provider by which `@erc725/erc725.js` will request blockchain data is set on
 
 The following provider types are supported:
 
-## Web3
+## RPC url
 
-The following code snippet will use the web3 provider available at web3.providers from the corresponding `web3` library.
+An rpc url can be passed to the configuration object.
 
 ```javascript
-import Web3 from 'web3';
-
-const web3provider = new Web3(
-  new Web3.providers.HttpProvider('https://rpc.l16.lukso.network'),
-);
+const rpcUrl = 'https://rpc.l16.lukso.network';
```

Comment:
Would be good to show how it is passed to config object here. I don't see an example of instantiation

Reply:
True. Only thing is that the others examples dont show it but i guess i can add it to all

Reply:
@Hugoo wdyt?

Reply:
@CallumGrindle - which config object? The RPC URL is not set in the configs but in place of the current provider.

Reply:
@Hugoo I meant how it is passed on instantiation.

Reply:
There is no example of how the provider is passed to the ERC725 object on this page